sqlite/src/func.c

```diff
@@ -2699,9 +2699,11 @@ void sqlite3RegisterBuiltinFunctions(void){
     FUNCTION2(unlikely,          1, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
     FUNCTION2(likelihood,        2, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
     FUNCTION2(likely,            1, 0, 0, noopFunc,  SQLITE_FUNC_UNLIKELY),
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
 #ifdef SQLITE_DEBUG
     FUNCTION2(affinity,          1, 0, 0, noopFunc,  SQLITE_FUNC_AFFINITY),
 #endif
+#endif /* defined(SQLITE_BUILDING_FOR_COMDB2) */
```

Comment:
You want to comment affinity() for comdb2 builds? If so, the above change wouldn't help.

Also, I think this change will break @mistachkin's merge script. You need something to the effect :
```
#if defined(SQLITE_BUILDING_FOR_COMDB2)
//#ifdef SQLITE_DEBUG
//    FUNCTION2(affinity,          1, 0, 0, noopFunc,  SQLITE_FUNC_AFFINITY),
//#endif
#else /* defined(SQLITE_BUILDING_FOR_COMDB2) */
#ifdef SQLITE_DEBUG
    FUNCTION2(affinity,          1, 0, 0, noopFunc,  SQLITE_FUNC_AFFINITY),
#endif
#endif /* defined(SQLITE_BUILDING_FOR_COMDB2) */
```

Reply:
yeah, you are absolutely right. I must have been daydreaming. Will fix, thanks for looking.

Reply:
so i think !defined is supported by the script because i see other usages that don't have the else part.